src/components/Clients/ClientListTable.jsx

```diff
@@ -56,20 +58,15 @@ const ClientListTable = ({ setSelectedClientToDelete, setShowDeleteClientModal }
           </TableRow>
         </TableHead>
         <TableBody>
-          {sortedUserList?.map((client, index) => {
-            const labelId = `clientlist-checkbox-${index}`;
-            return (
-              <ClientListTableRow
-                key={client.webId}
-                labelId={labelId}
-                client={client}
-                state={state}
-                dispatch={dispatch}
-                setShowDeleteClientModal={setShowDeleteClientModal}
-                setSelectedClientToDelete={setSelectedClientToDelete}
-              />
-            );
-          })}
+          {sortedUserList?.map((client, index) => (
+            <ClientListTableRow
+              key={client.webId}
+              labelId={index + 1}
```

Comment:
We can remove this property. The cell it was labeling has been deleted.

Reply:
Got this removed and in the children as well.